R/documentation.R

```diff
@@ -31,6 +31,10 @@ default_doc <- function() {
 #' Exported documentation helper function.
 #'
 #' @param internal Not used.
+#'
+#' @param index_features Positive integer vector. Specifies the indices of combinations to
+#' apply to the present method. `NULL` means all combinations. Only used internally.
+#'
```

Comment:
Note to Lars: Added this to default doc instead.

Reply:
Good, thank you. :) (Previously, I did not want to change that file without your consent, hence I tried to keep the changes locally, but now I know that I should have put it here)